src/Raven.Server/Documents/Handlers/Admin/AdminDatabaseRequestHandler.cs

```diff
@@ -1,14 +1,45 @@
-﻿using Raven.Server.Web;
+﻿using System;
+using System.Threading.Tasks;
+using Raven.Server.Routing;
+using Raven.Server.ServerWide.Context;
+using Raven.Server.Web;
+using Sparrow;
+using Sparrow.Json;
 
 namespace Raven.Server.Documents.Handlers.Admin
 {
+    
     public abstract class AdminDatabaseRequestHandler : DatabaseRequestHandler
     {
         // TODO : implement "admin" part
     }
 
     public abstract class AdminRequestHandler : RequestHandler
     {
+        
         // TODO : implement "sys admin" part
     }
+
+    public class RachisAdminHandler : AdminRequestHandler
+    {
+        private static readonly StringSegment TypeSegment = new StringSegment("Type");
+
+        [RavenAction("/rachis/apply", "POST", "/rachis/apply")]
+        public async Task ApplyCommand()
```

Comment:
I think that this should be leader only method, so it will fail if you are not the leader.
Other endpoints will forward it to this one, if needed, but if this is called, this should either be the leader or fail and the caller will need to figure it out.